charts/flyte/README.md

```diff
@@ -1,59 +1,38 @@
-# flyte
+# Flyte Sandbox Chart
```

Comment:
We don't edit readme directly, It will be generated by template that is available in same directory, After making your changes in template just run `make helm`